express-notes-api/index.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-console */
+const express = require('express');
+const app = express();
+const fs = require('fs');
+const data = require('./data.json');
+let stringifiedNoteData = null;
+
+app.use(express.json());
+
+function newNoteCreation(content) {
+  data.notes[data.nextId] = {};
+  data.notes[data.nextId].id = data.nextId;
+  data.notes[data.nextId].content = content;
+  data.nextId++;
+  return data;
+}
+
+app.get('/api/notes', function (req, res) {
+  const notes = [];
+  const notesKeyLength = Object.keys(data.notes).length;
+  if (notesKeyLength === 0) {
+    res.send(notes);
+  } else {
+    for (const i in data.notes) {
+      notes.push(data.notes[i]);
+    }
+    res.send(notes).json();
```

Comment:
We normally do this as `res.json(notes)`. `.send()` works, but it's better to use the more specific method. Please replace your `.send()` calls with `.json()`

Reply:
Okay, will do. I get mixed up with which one to use. Thanks.